WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -1139,14 +1140,14 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
     }
 
     func gutenbergMediaSources() -> [Gutenberg.MediaSource] {
-        workaroundCoreDataConcurrencyIssue(accessing: post) {
+        post.managedObjectContext?.performAndWait {
             [
                 post.blog.supports(.stockPhotos) ? .stockPhotos : nil,
                 post.blog.supports(.tenor) ? .tenor : nil,
                 .otherApps,
                 .allFiles,
             ].compactMap { $0 }
-        }
+        } ?? []
```

Comment:
Probably out of scope of this PR, but I wonder if it's okay to store the "media sources" during view controller's init method, because the `blog.supports` call results probably won't change while user is editing a post.